lib/signature.coffee

```diff
@@ -1,8 +1,18 @@
 _ = require('lodash')
 _.str = require('underscore.string')
+async = require('async')
 Parameter = require('./parameter')
 settings = require('./settings')
 parse = require('./parse')
+utils = require('./utils')
+
+isLastOne = (parameters, predicate) ->
+	index = _.findIndex(parameters, predicate)
```

Comment:
Can it use either of these:
If it needs to be a precise match: `return parameters[parameters.length - 1] is predicate`
If it should match the lodash functionality: `return _.findIndex(parameters[-1...], predicate) is 0`

Both of these will only check the very last parameter, saving the processing of the ones beforehand (although it does have the issue of not working in the case it has to only match for the very last one, rather than matching for the last one regardless of ones before it)